WindowsServerDocs/get-started-19/servicing-channels-19.md

```diff
@@ -75,7 +83,7 @@ You've likely already chosen to use at least one of these options based on your
 
 ## Where to obtain Semi-Annual Channel releases
 
-Semi-Annual Channel releases should be installed as a clean installation.
+Semi-Annual Channel releases should be installed as a clean installation. It is possible to use In-place upgrade via ISO from one SAC to a later version.
```

Comment:
Codestyle/capitalization: This needs to be coherent, either fully capitalized or fully lowercase, depending on whether the feature is the object in focus.
Since the feature "In-place Upgrade" is not the main focus, seeing as Semi-Annual Channel is the main focus, I suggest this:
```suggestion
Semi-Annual Channel releases should be installed as a clean installation. It is possible to use in-place upgrade via ISO from one SAC to a later version.
```